src/middleware/likernft.ts

```
@@ -36,7 +36,7 @@ export const fetchISCNPrefixes = async (req, res, next) => {
   try {
     const { class_id: classId } = req.query;
     if (!classId) throw new ValidationError('MISSING_ISCN_OR_CLASS_ID');
-    const classIds = Array.isArray(classId) ? classId : [classId];
+    const classIds = Array.isArray(classId) ? Array.from(new Set(classId)) : [classId];
```

prefer throwing error instead of silent remove